WooCommerce/WooCommerceTests/ViewRelated/Orders/Issue Refund/RefundCreationUseCaseTests.swift

```diff
@@ -86,4 +89,72 @@ final class RefundCreationUseCaseTests: XCTestCase {
         XCTAssertEqual(refund.items[0].taxes[1].taxID, 12)
         XCTAssertEqual(refund.items[0].taxes[1].total, "2.20")
     }
+
+    func test_refund_shipping_values_with_no_items_are_transformed_correctly() {
+        // Given
+        let shippingTaxes = ShippingLineTax(taxID: 2, subtotal: "", total: "0.99")
+        let shippingLine = ShippingLine(shippingID: 5, methodTitle: "", methodID: "", total: "7.00", totalTax: "0.99", taxes: [shippingTaxes])
+        let useCase = RefundCreationUseCase(amount: "7.99",
+                                            reason: nil,
+                                            automaticallyRefundsPayment: false,
+                                            items: [],
+                                            shippingLine: shippingLine,
+                                            currencyFormatter: formatter)
+
+        // When
+        let refund = useCase.createRefund()
+
+        // Then
+        XCTAssertEqual(refund.items.count, 1)
```

Comment:
nit: I was wondering if it's worth asserting the whole `items` array to `[OrderItemRefund(itemID: 5, ...)]` just in case we might start using some other fields in `OrderItemRefund`. Either way works though!